config/install/tripal.tripalfield_collection.trpcultivate_experiments.yml

```
@@ -917,20 +833,798 @@ fields:
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: SOSA
+            termAccession: FeatureOfInterest
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+
+    -   name: ghouse_objectives
+        content_type: greenhouse_experiment
+        label: Objectives
+        type: chado_property_type_default
+        description: The objectives of this experiment. These should be the questions you are trying to answer by conducting this experiment. For example, "Determine the magnitude of effect environment has on my trait of interest."
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: SIO
+            termAccession: '000337'
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+
+    -   name: ghouse_growingmedia
+        content_type: greenhouse_experiment
+        label: Growing Media
+        type: chado_property_type_default
+        description: Record the growing media being used in this controlled environment experiment (i.e. Vermiculite, Sand). Be specific with volume ratio (i.e. Peat, Perlite, Vermiculite 2:1:1)
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: rdfs
+            termAccession: type
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+              
+## Growth Chamber Experiment ##
+## Chado Table: project
+## Required Columns: name (varchar 255)
+## Optional columns: description (text)
+## Related: projectprop, project_pub, project_relationship
+##          array_project, nd_experiment_project, project_analysis, project_contact, project_dbxref, project_feature, project_stock
+
+    -   name: experiment_name
+        content_type: growthchamber_experiment
+        label: Name
+        type: chado_string_type_default
+        description: The name of the field experiment. It is recommended the name include important details such as siteyears, crop and germplasm collection.
```

"crop" looks awkward to me by itself. Maybe "crop species"?
```suggestion
        description: The name of the field experiment. It is recommended that the name include important details such as siteyears, crop and germplasm collection.
```

---

Sorry Carolyn, I completely missed you comments about changing "crop" to "crop species" 😅 have both of them changed now. Thank you for catching them!